### ISSUE TITLE
xeCJK package

### ISSUE THREAD

**Issue by jteresco** [lubaochuan/ccsc-editor#2]
I had to comment out the

`\usepackage{xeCJK}`

line to get the sample to compile on my Mac's installation (a standard and up-to-date texlive, I believe, installed through fink).  When I kept it in and used the xelatex command as the first errors suggested, I was getting many more errors about fonts that could not be installed.

Is there any harm in removing that package for the default template?  Once I did it, I was able to use pdflatex as I normally do.

**Comment by lubaochuan** [comment]
Nope. I included the library to support Chinese characters in a paper
published last year. This package is not required unless you need it.
Thanks for letting me know.
